content/en/infrastructure/livecontainers.md

```diff
@@ -274,20 +301,17 @@ You can see logs that you have chosen to index and persist by selecting a corres
 [1]: https://app.datadoghq.com/containers
 [2]: /integrations/docker_daemon/
 [3]: /agent/kubernetes/
-[4]: /integrations/amazon_ecs/
-[5]: https://app.datadoghq.com/containers/kubernetes-beta
-[6]: /agent/docker/#run-the-docker-agent
-[7]: /agent/kubernetes/?tab=helm
-[8]: /agent/
-[9]: /agent/cluster_agent/setup/?tab=secret
-[10]: /agent/kubernetes/?tab=helm#installation
-[11]: /getting_started/tagging/
-[12]: /getting_started/tagging/unified_service_tagging
-[13]: /dashboards/widgets/timeseries/
-[14]: /logs
-[15]: /metrics
-[16]: /tracing
-[17]: /events
-[18]: /logs/live_tail/
-[19]: https://gist.github.com/hkaj/404385619e5908f16ea3134218648237
-[20]: https://github.com/DataDog/helm-charts/tree/master/charts/datadog
+[4]: /agent/amazon_ecs/
+[5]: https://app.datadoghq.com/orchestration/overview
+[6]: /agent/
+[7]: /agent/cluster_agent/setup/
+[8]: /tagging/assigning_tags?tab=agentv6v7#host-tags
+[9]: /getting_started/tagging/unified_service_tagging
+[10]: /dashboards/widgets/timeseries/
+[11]: /logs
+[12]: /metrics
+[13]: /tracing
+[14]: /events
+[15]: /logs/live_tail/
+[16]: https://gist.github.com/hkaj/404385619e5908f16ea3134218648237
```

Comment:
@hkaj this points to nothing. Do you remember what we had there?

Reply:
+1, forgot to mention this yesterday: we had an RBAC doc we used to link here that was in the private beta doc for live containers, but looks like a 404 now.

Reply:
oh good catch, probably some RBAC example. Let's use this one instead: https://github.com/DataDog/datadog-agent/blob/7.23.1/Dockerfiles/manifests/cluster-agent/rbac.yaml